e2e/README.md

```diff
@@ -3,20 +3,13 @@ End-To-End Tests written with cypress.io
 
 To run these tests:
 
-1. Clone the repository
-`git clone git@github.com:pol-is/polis-cypress.git`
-`cd polis-cypress`
+1. Install npm dependencies:
+    * `cd e2e`
+    * `npm install`
 
-2. Install npm dependencies
-`npm install`
+2. For "headless" testing:
+    * `$(npm bin)/cypress run`
```

Comment:
I wasn't intending this to be the recommended way -- `npm test` or one of the other `npm run <foo>` scripts should be used. I'm avoiding wholesale `cypress run`, because we will have some tests that require third-party credentials like Google service account tokens, api keys, etc (there are already some in PRs :) )

Reply:
Ah, actually, just there's an outstanding PR where this readme is fixed. Lemme find it